oligo_designer_toolsuite/oligo_specificity_filter/_specificity_filters.py

```diff
@@ -0,0 +1,35 @@
+class SpecificityFilter:
+    """Thus class applies the specificity filters given in input sequentally to a ``CustomDB`` class.
+
+    :param filters: List of all the filter classes that we want to apply to the database
+    :type filters: list of ``SpecificityFilterBase`` class
+    """
+
+    def __init__(self, filters):
+        self.filters = filters
+
+    def apply(self, DB, n_jobs=None, write_empty_genes=True):
+        """Applies all the class filter sequentailly to the ``oligos_DB`` sotored in the DB class given in input. Each filter parallelized and ``n_jobs`` represents the maximum number of cores available.
```

Comment:
Applies all the class filter **sequentially** to the ``oligos_DB`` **stored** in the DB class...
:param n_jobs: maximum number of available jobs, if ``None`` **the** number of jobs of the ``DB`` class is used, defaults to None